drivers/filesystems/udfs/CMakeLists.txt

```diff
@@ -52,8 +52,8 @@ if(MSVC)
         add_target_compile_flags(udfs "-Wno-tautological-unsigned-zero-compare -Wno-self-assign")
         add_target_compile_flags(udfs "-Wno-sometimes-uninitialized -Wno-parentheses-equality")
     endif()
-else()
-    add_target_compile_flags(udfs "-Wno-unused-but-set-variable")
+elseif(GCC)
+    target_compile_options(udfs PRIVATE "-Wno-unused-but-set-variable;-Wno-stringop-overflow")
```

Comment:
`target_compile_options(udfs PRIVATE -Wno-unused-but-set-variable -Wno-stringop-overflow)` (or with two sets of quotes) works too, right? Seems more readable than the semicolon IMHO.